File: modules/expo-pose-camera/android/src/main/java/expo/modules/posecamera/ExposePoseCameraModule.kt
package expo.modules.posecamera

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class ExposePoseCameraModule : Module() {
    override fun definition() = ModuleDefinition {
        Name("ExposePoseCamera")

        View(PoseCameraView::class) {
            Events("onPoseDetected")

            Prop("isActive") { view: PoseCameraView, isActive: Boolean ->
                view.setActive(isActive)
            }
        }
    }
}

File: modules/expo-pose-camera/ios/ExposePoseCameraModule.swift
import ExpoModulesCore

public class ExposePoseCameraModule: Module {
    public func definition() -> ModuleDefinition {
        Name("ExposePoseCamera")

        View(PoseCameraView.self) {
            Events("onPoseDetected")

            Prop("isActive") { (view: PoseCameraView, isActive: Bool) in
                view.setActive(isActive)
            }
        }
    }
}
